packages/support/resources/views/components/badge.blade.php

```diff
@@ -55,9 +57,17 @@
         <x-filament::icon :icon="$icon" :class="$iconClasses" />
     @endif
 
-    <span>
-        {{ $slot }}
-    </span>
+    @if ($truncated)
+        <div class="grid">
+            <span class="truncate">
+                {{ $slot }}
+            </span>
+        </div>
+    @else
+        <span>
+            {{ $slot }}
+        </span>
+    @endif
```

Comment:
```suggestion
    <span class="truncate>
        {{ $slot }}
    </span>
```

Reply:
You'd think that this will work, except it won't because this component is used inside other components that have the `flex` class